dts/bindings/sensor/invensense,icm42670-common.yaml

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2024 TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+include: [sensor-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    required: true
+    description: |
+      The INT signal default configuration is active-high.  The
+      property value should ensure the flags properly describe the
+      signal that is presented to the driver.
+
+  power-mode:
+    type: string
+    default: "low noise"
```

Comment:
missing justification

Reply:
Thanks for the comment. I added the default value justification for this property also.